jerry-core/parser/js/js-parser.c

```diff
@@ -141,7 +141,8 @@ parser_compute_indicies (parser_context_t *context_p, /**< context */
     if (char_p != NULL)
     {
       literal_p->u.value = ecma_find_or_create_literal_string (char_p,
-                                                               literal_p->prop.length);
+                                                               literal_p->prop.length,
+                                                               (literal_p->status_flags & LEXER_FLAG_ASCII));
```

Comment:
I think the result is boolean, so this should be:
`(literal_p->status_flags & LEXER_FLAG_ASCII) != 0`
Similar cases below.

Come to think of it, this flag could be the first bit, so the compiler does not need to do any conversion.